README.template.md

```diff
@@ -1,14 +1,30 @@
+<!-- This is the readme template for commercetools nodejs projects. We recommend you following this format when writing a readme project -->
+<!-- Commercetools Logo -->
+![commercetools-icon](https://github.com/commercetools/press-kit/blob/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png?raw=true)
+
 # Unslugified Project Title
+<!-- Badges -->
 [![Travis Build Status][travis-icon]][travis]
 [![Codecov Coverage Status][codecov-icon]][codecov]
 [![David Dependencies Status][david-icon]][david]
 [![David devDependencies Status][david-dev-icon]][david-dev]
 
 A short description of what this project is about.
 
+<!-- Getting started section -->
+## Getting started
+
 ## Usage
 How to use this project, possibly with a small code example.
 
+<!-- Contributing section. You should have a CONTRIBUTING.md file in your project -->
+## Contributing
+See [CONTRIBUTING.md](CONTRIBUTING.md) for info on contributing to eslint-config
+
+<!-- License section. You should have a license file in your project. We really our opensource projects under the MIT-LICENSE -->
+## License
```

Comment:
I would say leave both the contributing and license part out, I think it only adds obvious information. Especially the for  the license since GitHub now also shows it in the top bar of the project.